doc/qbittorrent.1.md

```diff
@@ -1,36 +1,236 @@
 % QBITTORRENT(1) Bittorrent client written in C++ / Qt
-% Christophe Dumez <chris@qbittorrent.org>
-% January 16th 2010
+%
+% 2020-03-20
 
 # NAME
-qBittorrent - a Bittorrent client written in C++ / Qt
 
+**qBittorrent** - a BitTorrent client written in C++ / Qt
 
 # SYNOPSIS
-**qbittorrent** `[--no-splash] [--webui-port=x] [TORRENT_FILE | URL]...`
-
-**qbittorrent** `--help`
-
-**qbittorrent** `--version`
 
+| **qbittorrent** \[_OPTIONS_\] \[_URI_...\]
+| **qbittorrent** **`-h` | `-v`**
 
 # DESCRIPTION
+
 **qBittorrent** is an advanced Bittorrent client written in C++ / Qt,
-using the **libtorrent-rasterbar** library by Arvid Norberg. qBittorrent is similar to uTorrent.
-qBittorrent is fast, stable, light, it supports unicode and it provides a good integrated
-search engine. It also comes with UPnP port forwarding / NAT-PMP, encryption (Vuze compatible),
-FAST extension (mainline) and PeX support (utorrent compatible).
+using the **libtorrent-rasterbar** library by Arvid Norberg as the backend. qBittorrent provides a familiar GUI similar to uTorrent's, and is fast, stable and light.
+
+qBittorrent supports several important modern features such as UPnP port forwarding / NAT-PMP, kademelia DHT (Mainline) and magnet link support, IPv6, uTP, PeX, LSD, UDP tracker protocol support and more. Refer to the **libtorrent-rasterbar** documentation for the full list of features and supported BEPs.
+
+In addition, qBittorrent supports Unicode and it provides a good integrated Python-based torrent search engine, an RSS downloader, a torrent creation GUI, a basic embedded tracker to facilitate sharing torrents with friends, and an HTTP API/Web UI combo for remote access/administration.
+
+When qBittorrent is started from a terminal, its **stdout** and **stderr** will be attached to that terminal.
+
+If no arguments are specified, qBittorrent simply starts and shows the GUI.
+
+If either **-h** or **-v** is specified, the relevant information is printed to the terminal and qBittorrent exits. Passing any other option alongside **-h** or **-v** is not allowed.
+
+_URI_ is a string pointing to a BitTorrent metainfo file (a "torrent"). It may be a path to a file on disk or an online resource accessible through a supported URI scheme such as **magnet**, **HTTP**, or **HTTPS**.
+
+Standalone **info-hashes** are also accepted as shorthand for a bare-bones **magnet** _URI_ constructed from that **info-hash**. For example, passing the **info-hash** _`e73108cbd628fee5cf203acdf668c5bf45d07810`_ is the same as passing the **magnet** _URI_ _`magnet:?xt=urn:btih:e73108cbd628fee5cf203acdf668c5bf45d07810`_.
```

Comment:
Lets add a note about that info hash is for "ubuntu-18.04.4-live-server-amd64.iso"

Reply:
FIxed.